src/interfaces/HposInterface.ts

```diff
@@ -639,6 +668,48 @@ export function useHposInterface(): HposInterface {
     }
   }
 
+  async function redeemHoloFuel(payload: RedeemHoloFuelPayload): Promise<Transaction | boolean> {
+    try {
+      const getReserveDetailsParams = {
+        appId: localStorage.getItem(kCoreAppVersionLSKey),
+        roleId: 'holofuel',
+        zomeName: 'reserves',
+        fnName: 'get_all_reserve_accounts_details',
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        payload: null
+      }
+
+      const reserveAccountsDetails: ReserveSettingsResponse = await hposHolochainCall({
+        method: 'post',
+        path: '/zome_call',
+        params: getReserveDetailsParams
+      })
+
+      const initiateRedemptionParams = {
+        appId: localStorage.getItem(kCoreAppVersionLSKey),
+        roleId: 'holofuel',
+        zomeName: 'transactor',
+        fnName: 'redemption',
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        payload: {
+          reserve_id: reserveAccountsDetails.hash,
```

Comment:
```suggestion
          reserve_id: reserveAccountsDetails.reserve_id,
```

Updating to match changes to implementation.